### PR TITLE
Add meta fields with required flag on RenderForm

### PR DESCRIPTION
**Pull request by NerdsvilleCEO**

### CODE REVIEW

File: templatefunc.go [hunk 1]
```diff
@@ -496,6 +502,21 @@ func parseFormTag(fieldT reflect.StructField) (label, name, fType string, id str
 			label = tags[2]
 		}
 	}
+
+	return
```
Comment (astaxie) [inline templatefunc.go]: why add this return?

File: templatefunc.go [hunk 2]
```diff
@@ -422,17 +422,18 @@ func RenderForm(obj interface{}) template.HTML {
 		fieldT := objT.Field(i)
 
 		label, name, fType, id, class, ignored := parseFormTag(fieldT)
+		required := parseMetaTag(fieldT)
```
Comment (astaxie) [inline templatefunc.go]: I also suggest add the Meta into `parseFormTag`

File: templatefunc.go (hunk 1 above)
Comment (NerdsvilleCEO) [inline templatefunc.go]: Looks like the diff is backwards lol, must be a github issue, I don't even see this locally, but look at the entire file around these lines, the end result should be good

File: templatefunc.go (hunk 2 above)
Comment (NerdsvilleCEO) [inline templatefunc.go]: Fixed :)

File: templatefunc.go [hunk 3]
```diff
@@ -470,6 +475,15 @@ func parseFormTag(fieldT reflect.StructField) (label, name, fType string, id str
 	id = fieldT.Tag.Get("id")
 	class = fieldT.Tag.Get("class")
 
+	meta := strings.Split(fieldT.Tag.Get("meta"), ",")
```
Comment (astaxie) [inline templatefunc.go]: I suggest just use `required`, use `meta` is confuse for me

Comment (NerdsvilleCEO) [inline templatefunc.go]: I changed this, also I covered some edge cases. Please review now :) Thank you for considering this change

**Comment by NerdsvilleCEO** [comment]
@astaxie Can you try re-running this build? I see some sort of TCP error in the log

**Comment by NerdsvilleCEO** [comment]
Hi @astaxie any update on this

**Comment by astaxie** [comment]
@NerdsvilleCEO thanks for the update

**Pull request merged**